docs/faq.rst

```diff
@@ -451,7 +451,7 @@ For example, if you run the command:
    ispc foo.ispc -o foo.o --target=sse2,sse4-x2,avx-x2
 
 Then four object files will be generated: ``foo_sse2.o``, ``foo_sse4.o``,
-``foo_avx.o``, and ``foo.o``.[#]_  Link all of these into your executable, and
```

Comment:
This change is really about inserting a space before `[#]_` so it will be parsed correctly. Moving the `.` is just a minor style change that could be left out.